src/views/Accessibility/index.tsx

```diff
@@ -113,7 +137,7 @@ const Accessibility = () => {
   if (isUserSet) {
     if (user.isAccessibilityTeam(userGroups, flags)) {
       return (
-        <PageTemplate>
+        <PageTemplate surveyUrl={REPORT_PROBLEM_ACCESSIBILITY_TEAM_SURVEY}>
```

Comment:
```suggestion
        <PageTemplate isAccessibilityTeam>
```

This is what I was leaning towards. There are drawbacks to your implementation and my suggestion. IMO, the header, footer, etc should be composed on each page.